Fix aclocal warning:
/usr/share/aclocal/gdk-pixbuf.m4:12: warning: underquoted definition of AM_PATH_GDK_PIXBUF
--- gdk-pixbuf-0.22.0/gdk-pixbuf.m4
+++ gdk-pixbuf-0.22.0/gdk-pixbuf.m4
@@ -11,3 +11,3 @@
 dnl
-AC_DEFUN(AM_PATH_GDK_PIXBUF,
+AC_DEFUN([AM_PATH_GDK_PIXBUF],
 [dnl 
